File: src/main/java/com/sica/backend/dto/AccessRecordResponse.java
package com.sica.backend.dto;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.LocalTime;

public record AccessRecordResponse(
    String id,
    String usuarioId,
    String usuarioNombre,
    String usuarioFoto,
    String tipo,
    LocalDate fecha,
    LocalTime hora,
    String dispositivo,
    String ubicacion,
    String metodo,
    LocalDateTime creadoEn
) {}

File: src/main/kotlin/com/sica/backend/dto/AccessRecordResponse.kt
package com.sica.backend.dto

import java.time.LocalDate
import java.time.LocalDateTime
import java.time.LocalTime

data class AccessRecordResponse(
    val id: String,
    val usuarioId: String,
    val usuarioNombre: String,
    val usuarioFoto: String?,
    val tipo: String,
    val fecha: LocalDate,
    val hora: LocalTime,
    val dispositivo: String,
    val ubicacion: String,
    val metodo: String,
    val creadoEn: LocalDateTime
)
